OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -267,27 +253,36 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
     */
     virtual void scale(const ScaleSet& aScaleSet);
 
-    /**
-    * ModelComponent display interface.
-    *
-    * This method appends the visuals for the Joint to the list appendToThis.
-    * Base class adds geometry for the two Frames making the joint, any extra 
-    * gometry needed to visualize the Joint would be added by the subclass.
+protected:
+    /** A Coordinate place-holder created by constructCoordinate(). E.g.:  
```

Comment:
Clarify that you are talking about a new joint class? Otherwise I thought you were suggesting the use of a nested class.